imsim/camera.py

```diff
@@ -160,14 +181,31 @@ class Camera(dict):
     Class to represent the LSST Camera as a dictionary of CCD objects,
     keyed by the CCD name in the focal plane, e.g., 'R01_S00'.
     """
-    def __init__(self, camera_class='LsstCam'):
+    _req_params = {'camera': str}
```

Comment:
I think these aren't being used/checked here.  Maybe they belong with the others in the CameraReadout class?